site/pages/our-work/case-study/what-to-read-next/slice.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+import React from 'react';
+import InlineSVG from 'svg-inline-react';
+
+import styles from './style.css';
+import arrowSVG from '../../../../../assets/images/SVG/arrow.svg';
+
+type WhatToReadNextSliceProps = {
+  details: Object,
+};
+
+const WhatToReadNextSlice = ({ details }: WhatToReadNextSliceProps) => {
```

Comment:
UPDATE: This is sadly not possible as the spread operator is not supported in this version of node.